src/api/middleware/controllers/review.js

```diff
@@ -1,11 +1,52 @@
-const Review = require("../../../models/Review");
+const Review = require("../../../models/review");
 const { isValidObjectId } = require("../../../utils.js/validator");
 
 module.exports = {
   create: async (req, res) => {
-    const { reviewText, stars, likes, user } = req.body;
+    const { reviewText, stars, likes, user, book } = req.body;
 
     if (!isValidObjectId(user))
       return res.json({ message: "user is invalid " });
+    if (!isValidObjectId(book))
+      return res.json({ message: "book is invalid " });
+    if (!isValidObjectId(likes))
+      return res.json({ message: "likes is invalid " });
```

Comment:
`Likes` ought to have been an array so that many users can like or dislike a particular review but from your model, it is not. So update you model  to look like below then remove this validation on `likes`, also remove the validation for `likes` in `review.validator.js`
```javascript
likes: [{
      type: Schema.Types.ObjectId,
      ref: 'user',
    }],
```